README.md

```diff
@@ -48,18 +48,17 @@ The [scripts folder](./scripts/) provides utilities to enable the backend and it
 Check the documentation in the comments at the top of each file for usage instructions.
 
 
-# Deploying under a domain name using nginx-proxy
+# Deploying under a domain name using swag-proxy
 In order to make the application's docker container accessible under a public URL, you need to expose it using a tool of your choice.
-The system works well with the [nginx-proxy](https://github.com/nginx-proxy/nginx-proxy) docker. This allows to automatically configure things so that the app is reachable under a specific domain name (including automatic setup of SSL certificates through letsencrypt).
```

Comment:
let's delete the whole nginx-proxy folder then?

Reply:
its still in use on the prod-1 - i will remove this, as soon this is not used anymore